src/services/tags/delete-tag.test.js

```diff
@@ -1,18 +1,24 @@
 const {expect} = require('chai')
 const db = require("../../db")
 const deleteTag = require("./delete-tag")
-const {fixtures} = require('../../../test/utils')
+const {fixtures, resetDBTable} = require('../../../test/utils')
 
 describe('DELETE tag', () => {
+
 	let tag
+
 	before(async ()=>{
+		await resetDBTable('tags')
```

Comment:
please could you shed light on the reason for this change?

Reply:
multiple tags ended up having the same tagId which passed locally but wasn't passing on github action. Hence the fix was to clear the tags table each time

Reply:
okay, perhaps you're referring to something else but multiple tags can't have the same tagId

Reply:
not sure where i asked for a change, we were having a discussion

Reply:
also if you've decided to revert it, we shouldn't have the file in the file changes then

Reply:
yes please